### ISSUE TITLE
test model

### ISSUE THREAD

**Issue by mumuZhao** [JetBrains-Research/authorship-detection#7]
Hello. We want to test the model generated by the training, but the test set is not tokens.csv and paths.csv comparison with training set tokens.csv and paths.csv. The number of token and path is completely different, so the generated model can't be used many times and can't be tested many times. Can you share the source program(attribution/pathminer/extract-path-contexts.jar) for generating these CSV files? We want to make some adjustments to the program to achieve the reuse of the model. thank you.
